po/uk.po

```diff
@@ -1029,85 +1020,64 @@ msgstr "Пресет %(number)d: %(title)s"
 
 #: picard/script/__init__.py:195
 msgid "Default file naming script"
-msgstr "Стандартний скрипт іменування файлів"
+msgstr "Скрипт іменування файлів за замовчуванням"
 
 #: picard/script/__init__.py:207
 msgid "[album artist]/[album]/[track #]. [title]"
-msgstr "[виконавець альбому]/[альбом]/[трек #]. [назва]"
+msgstr "[album artist]/[album]/[track #]. [title]"
 
 #: picard/script/__init__.py:219
 msgid "[album artist]/[album]/[disc and track #] [artist] - [title]"
-msgstr "[виконавець альбому]/[альбом]/[# диска та треку] [виконавець] - [назва]"
+msgstr "[album artist]/[album]/[disc and track #] [artist] - [title]"
```

Comment:
Translating the descriptions here is fine. This is shown to the user as a help to understand how the file names will be constructed. So it is helpful to show it in native language.